dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumAttributes.kt

```diff
@@ -84,6 +84,24 @@ object RumAttributes {
      */
     const val ERROR_RESOURCE_URL: String = "error.resource.url"
 
+    /**
+     * The error message in case an Error was reported during the lifetime of a Span.
+     * This value is filled automatically by the [RumMonitor]
+     */
+    const val SPAN_ERROR_MESSAGE: String = "span.error_message"
+
+    /**
+     * The error type in case an Error was reported during the lifetime of a Span.
+     * This value is filled automatically by the [RumMonitor]
+     */
+    const val SPAN_ERROR_TYPE: String = "span.error_type"
+
+    /**
+     * The error stack in case an Error was reported during the lifetime of a Span.
+     * This value is filled automatically by the [RumMonitor]
+     */
+    const val SPAN_ERROR_STACK: String = "span.error_stack"
+
```

Comment:
Why do you need this ? A RUM Error already have those fields, so you should reuse them.

Reply:
I didn't find any of these fields in the RumAttributes. The reason we're adding these is to be consistent with the iOS SDK as they are sending the Span error information in separated/specific fields.

Reply:
I created `RUMM-679 Span error attributes should be send as RUMError values` to update this on iOS. 

Speaking of those 3 attributes:
* `span.error_message` → could become the `errorEvent.error.message` following the `rum-events-format`,
* `span.error_stack` → should become the `errorEvent.error.stack` following the `rum-events-format`, 
* `span.error_type` → 🚫 we don't have a good `errorEvent.error.*` value to map it to.

Maybe the `span.error_type` should be used along with `error_message` to build the `RUMError` message? So, instead of:
```swift
rumError.message = "Span {span.operationName} reported an error"
```
we would have:
```swift
rumError.message = "Span error ({span.operationName}): {SPAN_ERROR_TYPE} | {SPAN_ERROR_MESSAGE}"
```
where the `{SPAN_ERROR_TYPE } |` is an optional thing. WDYT @mariusc83 @xgouchet ?

Reply:
I think they should both be optional. Right now I don't have any guarantee that any of them is sent all the time.